checkpoint_quantum_firewall/manifest.json

```
@@ -0,0 +1,102 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "4b6b8ef9-e079-4ee4-877e-8f4aafbf8a1d",
+  "app_id": "checkpoint-quantum-firewall",
+  "display_on_public_website": true,
```

```suggestion
  "display_on_public_website": false,
```

https://github.com/DataDog/integrations-core/pull/16499/files#r1449975184

---

Done